tests/Fixer/Phpdoc/PhpdocScalarFixerTest.php

```diff
@@ -298,5 +298,11 @@ public static function provideFixCases(): iterable
             '<?php /** @var array{index: int, isRequired: bool} */',
             '<?php /** @var array{index: integer, isRequired: boolean} */',
         ];
+
+        yield [
```

Comment:
please add all 3 variants - https://phpstan.org/writing-php-code/phpdoc-types#bottom-type

Reply:
Added the two missing variants.